.circleci/config.yml

```diff
@@ -17,6 +17,7 @@ jobs:
       - run:
           name: Run baseline tests
           command: |
+            sudo npm link
```

Comment:
why do we need it?

Reply:
this links the bin file to the js, without this circleCI complains about not finding `gapic-generator-typescript`. But not sure if there's other way around.

Reply:
OK, that sounds good! I did not know `npm link` does that.